doc/classes/Image.xml

```diff
@@ -358,6 +358,7 @@
 			<param index="0" name="buffer" type="PackedByteArray" />
 			<description>
 				Loads an image from the binary contents of a JPEG file.
+				[b]Note:[/b] This method is only available in engine builds with the JPEG module enabled. By default, the JPEG module is enabled, but it can be disabled at build-time using the [code]module_jpg_enabled=no[/code] SCons option.
```

Comment:
```suggestion
				[b]Note:[/b] This method is only available in engine builds with the JPEG module, which is enabled by default. It can be disabled at build-time using the [code]module_jpg_enabled=no[/code] SCons option.
```
Or similar? For the rest as well...?